monitoring/.gitignore

```diff
@@ -0,0 +1,50 @@
+*.swp
```

Comment:
when consuming vendored code like this, can we link to that at top of file
e.g. https://github.com/AudiusProject/audius-protocol/blob/master/libs/src/services/web3Manager/Web3Manager.ts#L278

Reply:
Added a note